src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,31 @@
+.stars {
+  display: flex;
+  width: 100%;
+  margin-bottom: 22px;
```

Comment:
The value '22px' for margin-bottom seems to be a magic number. Consider defining it as a variable in 'variables.scss' for better maintainability, especially if this value is or will be reused in other places.